models/DataObject/ClassDefinition.php

```diff
@@ -1326,4 +1328,43 @@ public function getLinkGenerator()
 
         return $generator;
     }
+
+    /**
+     * @return bool
+     */
+    public function getCacheRawRelationData(): bool
+    {
+        return $this->cacheRawRelationData;
+    }
+
+    /**
+     * @param bool $cacheRawRelationData
+     *
+     * @return $this
+     */
+    public function setCacheRawRelationData($cacheRawRelationData)
+    {
+        $this->cacheRawRelationData = (bool) $cacheRawRelationData;
+
+        return $this;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getImplementsInterfaces(): ?string
+    {
+        return $this->implementsInterfaces;
+    }
+
+
+    /**
+     * @param string|null $implementsInterfaces
+     * @return $this
```

Comment:
Code style:
```suggestion
     *
     * @return $this
```

Reply:
real showstopper :-)

Reply:
Consistent code style makes code easier to read and reading it what we mostly do with code ;)